src/plugins/irc-events/unhandled.js

```diff
@@ -14,6 +15,7 @@ module.exports = function(irc, network) {
 		network.channels[0].pushMessage(client, new Msg({
 			type: Msg.Type.UNHANDLED,
 			command: command.command,
+			commandslug: slugify(command.command),
```

Comment:
Yes, but this would be solved by removing characters we don't want.

1. Replace space to `-`
2. Remove `/[^a-z0-9\-]/`